app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
+
+  private :update_likes_counter
+end
```

Comment:
- [ ] [OPTIONAL] Just as I have suggested earlier, I would like to suggest that you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_likes_counter method every time a new post has been saved to the database, then make the update_likes_counter method private.

Reply:
Thanks for the suggestion. I will study it a bit and try to implement in the next project's phase.

Reply:
Great 👍🏽